tests/saw/spec/handshake/handshake_io_lowlevel.saw

```diff
@@ -272,19 +272,23 @@ let s2n_advance_message_spec = do {
     let conn' = {{ advance_message conn }};
     crucible_ghost_value corked {{ conn'.corked }};
     crucible_points_to (conn_corked_io pconn) (crucible_term {{ conn'.corked_io }});
-    crucible_points_to (conn_mode pconn) (crucible_term {{ (advance_message conn).mode }});
-    crucible_points_to (conn_handshake_handshake_type pconn) (crucible_term {{ (advance_message conn).handshake.handshake_type }});
-    crucible_points_to (conn_handshake_message_number pconn) (crucible_term {{ (advance_message conn).handshake.message_number }});
+    crucible_points_to (conn_mode pconn) (crucible_term {{ conn'.mode }});
```

Comment:
good catch here 👍

Reply:
Agreed, although we can think about which property is more useful here. This is a spot where we could put in an additional property that we care about, to be sure we didn't somehow make the same mistake twice. So if it's a property of ```s2n_advance_message``` that the mode doesn't change we could also add:

```
crucible_postcond {{ conn.mode == conn'.mode }}
```

Reply:
I'm not actually changing the postconditions in this proof, just replacing the duplicate "advance_message conn"s with the equivalent "conn' ".